docs/docs/installation/docker-compose.mdx

```diff
@@ -15,7 +15,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 :::caution
 Since `docker compose` is primarily designed to run a set of containers on **a single host**
 and can't support requirements for **high availability**, we do not support nor recommend
-using our `docker- ompose` constructs to support production-type use-cases. For single host
+using our `docker-compose` constructs to support production-type use-cases. For single host
```

Comment:
`docker-compose` is being deprecated 😄 
```suggestion
using our `docker compose` constructs to support production-type use-cases. For single host
```

Reply:
@mistercrunch [I already did that](https://github.com/apache/superset/pull/30030)... and it was my fat fingers that screwed up this particular instance ;)